src/settings/MappingProfiles/MappingProfilesContainer/MappingProfilesContainer.js

```diff
@@ -77,8 +90,38 @@ const MappingProfilesContainer = ({
 
 MappingProfilesContainer.propTypes = {
   match: matchShape.isRequired,
+  mutator: PropTypes.shape({
+    mappingProfiles: PropTypes.shape({
+      GET: PropTypes.func.isRequired,
+    }),
+  }),
   history: historyShape.isRequired,
   location: locationShape.isRequired,
 };
 
-export default MappingProfilesContainer;
+MappingProfilesContainer.manifest = Object.freeze({
+  initializedFilterConfig: { initialValue: false },
+  query: { initialValue: {} },
+  resultCount: { initialValue: 5000 },
+  mappingProfiles: {
+    type: 'okapi',
+    path: 'data-export/mappingProfiles',
+    records: 'mappingProfiles',
+    recordsRequired: '%{resultCount}',
+    clientGeneratePk: false,
```

Comment:
Could you please check whether it makes sense adding perRequest: RESULT_COUNT_INCREMENT, option as well? I see some of the apps are using it and wondering what perRequest value would be if we have not provided this option.

Reply:
Thank you for paying attention to this. This option is responsible for adding the `limit` query parameter, limit the number of elements returned in the response.